src/sql/common/telemetryKeys.ts

```diff
@@ -42,3 +41,37 @@ export const Accounts = 'Accounts';
 export const FireWallRule = 'FirewallRule';
 export const AutoOAuth = 'AutoOAuth';
 export const AddNewDashboardTab = 'AddNewDashboardTab';
+
+// SQL Agent Events:
+
+// Views
+export const JobsView = 'JobsViewOpened';
+export const JobHistoryView = 'JobHistoryViewOpened';
+export const JobStepsView = 'JobStepsViewOpened';
+
+// Dialogs
+export const NewJobDialog = 'NewJobDialogOpened';
```

Comment:
For the Wizards and Actions, is there a way to add the instrumentation into the framework so we get usage metrics for all features generically?

Reply:
I can implement that. I agree it will make this much easier for all the features.